src/pybind/mgr/tox.ini

```
@@ -57,8 +57,10 @@ commands =
     mypy --config-file=../../mypy.ini \
            -m alerts \
            -m cephadm \
+           -m crash \
            -m dashboard \
            -m devicehealth \
+           -m hello \
```

Missing? The `fix` environment included `insights`, but not the `mypy`.
```suggestion
           -m hello \
           -m insights \
```

---

i haven't added the type annotations to the `insights` module yet.